posts/views.py

```diff
@@ -384,6 +399,43 @@ def get(self, request, pk):
 
         return render(request, 'post/post.html', context={'post': serializer.data, 'post_content': post_content, 'comments': serializer.data["comments"], "owns_post": owns_post})
 
+
+class FrontEndUserEditView(TemplateView):
```

Comment:
Please move this to a views file in the views folder. We can to limit the size of our views.py to make it easier to work with.

Reply:
Not going to worry about this for now. I just want this merged.